gui/create-item.lua

```diff
@@ -448,12 +450,12 @@ function hackWish(unit)
     if df.item_type.attrs[itemtype].is_stackable then
      createItem({mattype,matindex},{itemtype,itemsubtype},quality,unit,description,amount)
     else
-     local isCorpsePiece = itemtype == df.item_type.CORPSEPIECE
+     local isCorpsePiece = itemtype ~= df.item_type.CORPSEPIECE or itemtype ~= df.item_type.CORPSE
```

Comment:
Isn't this always true? A neq B or A neq C is always true unless A is B is C

Reply:
ah fuck wrong comparison type